src/cmd-kola

```diff
@@ -38,9 +38,17 @@ if os.path.isfile(blacklist_path):
     with open(blacklist_path) as f:
         blacklist = yaml.safe_load(f)
         for obj in (blacklist or []):
-            print(f"⚠️  Skipping kola test pattern \"{obj['pattern']}\":")
-            print(f"⚠️  {obj['tracker']}")
-            blacklist_args += ['--blacklist-test', obj['pattern']]
+            skiptest = False
+            # if there are any arches specified, skip tests only for those arches. if not skip unconditionally.
+            if 'arches' in obj:
+                if basearch in obj['arches']:
+                    skiptest = True
+            else:
+                skiptest = True
+            if skiptest:
```

Comment:
How about: `if basearch in obj.get('arches', [basearch]):` ?

Reply:
ah..nice! thanks

Reply:
We don't need `skiptest` at all anymore though, right? We could just put that same condition here.

Reply:
my bad...yes..removed